packages/components/src/components/text/text.jsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React from 'react';
+
+const Text = ({ children, size, color, align, weight, lineHeight, className }) => (
+    <>
+        <p
+            className={classNames('text', className, {
```

Comment:
Could we get these config values (class names) from a separate config file? For example, `text-config.js` ?